Core/MemoryFile.cpp

```diff
@@ -212,9 +212,16 @@ void MemoryFile::reloadFromFile() {
         SCOPED_LOCK(&lock);
 
         mmkv::getFileSize(m_diskFile.m_fd, m_size);
+        auto adjustSize = static_cast<size_t>(static_cast<double>(expectedCapacity) * 0.1f);
+        // 8 bytes header + 5 bytes key length + 5 bytes value length + 5 bytes value holder length + 4 bytes expire
+        expectedCapacity += std::max<size_t>(27, adjustSize);
```

Comment:
Is this adjustment really necessary? The users should know what they are doing. The `expectedCapacity` they passed in should be a multiply of `DEFAULT_MMAP_SIZE`.

Reply:
OK. If the value provided is a multiply of DEFAULT_MMAP_SIZE, it should have no problem here